web-client/src/theme/variables.scss

```diff
@@ -32,7 +32,7 @@
   // UI Kit: Primary 2 - Portland Orange
   --ion-color-primary: #ff642e;
   --ion-color-primary-rgb: 255, 100, 46;
-  --ion-color-primary-contrast: #000000;
+  --ion-color-primary-contrast: #ffffff;
```

Comment:
Is there a reason for deviating from the Ionic color generator's contrast color here?

If so, we should document it?

Reply:
This contrast match the designs, so yep, we should document it for now. Also not sure what the contrast ratio is here, if it is not great we should raise this with Abby